fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -366,6 +374,18 @@ public Date getLastModifiedDate() {
         return null;
     }
 
+    /**
+     * Set the last-modified date to the current date.
+     */
+    @VisibleForTesting
+    public void touch() {
+        try {
+            getNode().setProperty(FEDORA_LASTMODIFIED, Calendar.getInstance(getTimeZone("UTC")));
+        } catch (final RepositoryException e) {
+            LOGGER.info("Exception caught when trying to update lastModified date", e);
```

Comment:
Also, please do not dump the full exception (i.e. `LOGGER.info("...", e)`). Instead, just log the `e.getMessage()`.